source/less/blocks/cat-food.less

```
@@ -0,0 +1,37 @@
+.cat-food__wrapper {
+  background-color: #68b738d9;
+  background-image: url("../../img/mobile/main-image.png");
```

```suggestion
  background-image: url("../img/mobile/main-image.png");
```
Иначе браузер не найдёт файл изображения.